jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletChannel.java

```diff
@@ -78,6 +79,14 @@ public class ServletChannel
     private Request _request;
     private Response _response;
     private Callback _callback;
+    private CompleteListener _completeListener;
+    private final AtomicInteger _completeCount = new AtomicInteger(2);
```

Comment:
This is an insufficient mechanism as two write errors will count this down twice